go.mod

```diff
@@ -1,34 +1,49 @@
 module github.com/KyberNetwork/ethrpc
 
-go 1.21
+go 1.22
```

Comment:
I think using Go v1.22.7 is a good option, as the new version fixes a security bug

Reply:
Thanks, I've upgraded